Tasks/DownloadBuildArtifactsV0/task.json

```diff
@@ -198,7 +198,7 @@
             "resultSelector": "jsonpath:$.value[?(@.quality=='definition')]",
             "parameters": {
                 "project": "$(project)",
-                "name": "$(name)"
+                "name": "$(searchText)"
```

Comment:
This may be an issue with TFS AT DT mismatch case. Ensure that the UI is still sending both name and searchText.